Doc/library/stdtypes.rst

```diff
@@ -4822,33 +4822,53 @@ Generic Alias Type
    object: GenericAlias
    pair: Generic; Alias
 
-``GenericAlias`` objects are created by subscripting a class (usually a
-container), such as ``list[int]``.  They are intended primarily for
+``GenericAlias`` objects are generally created by
+:ref:`subscripting <subscriptions>` a class. They are most often used with
+:ref:`container classes <sequence-types>`, such as :class:`list` or
```

Comment:
Container classes can both sequence and mapping.

Reply:
Indeed. But the link reference for container classes in general (discussing both sequences and mappings) is `sequence-types`.

Reply:
This is where the `sequence-types` reference links to https://docs.python.org/3/reference/datamodel.html#emulating-container-types